docs/installation.rst

```diff
@@ -145,14 +153,26 @@ For example:
       # 1. Change the command to run
         command: yarn run debug
         volumes:
-          - "./config/config.json:/home/bookbrainz/bookbrainz-site/config/config.json:ro"
       # 2. Mount the src directory
           - "./src:/home/bookbrainz/bookbrainz-site/src"
+
+Ideally you save this new Compose file inside the ``local/`` directory, which will be ignored by git, for example as ``local/docker-compose.live-reload.yml``.
```

Comment:
This change still has to be done in the `.gitignore` of `bookbrainz-site` of course.
I haven't done this myself, as you said that you will do it yourself.

Reply:
Done !